rest-api/dao/questionnaire_response_dao.py

```diff
@@ -228,6 +229,15 @@ def _update_participant_summary(
         participant_summary.race = race
         something_changed = True
 
+    # if primary language provided in the response, set the new value.
+    resource_json = json.loads(questionnaire_response.resource)
+    for extension in resource_json.get('extension', []):
+      if extension['url'] == _LANGUAGE_EXTENSION:
+        if participant_summary.primaryLanguage != extension['valueCode']:
```

Comment:
I'm not sure whether we should change primaryLanguage on participant summary  to match the latest questionnaire response, or just for the initial consent. We should inquire.